### PR TITLE
add sql scripts for creating DB and inserting values to it

### PR DESCRIPTION
**Pull request by vladvorontsov**

### CODE REVIEW

**Comment by AnastasiaBlack** [comment]
Thank you very much for the scripts:) I have a question: we've discussed using unique constraint for the column 'nickname', what do you think about that? We need it for security purposes, maybe we should find a way to make this column unique?Thanks)

**Comment by thedikiy** [comment]
Have you tried to run your insert script? it' not working for me. 
1452: Cannot add or update a child row: a foreign key constraint fails (`сeasar`.`role`, CONSTRAINT `fk_role_category1` FOREIGN KEY (`category_id`) REFERENCES `ceasar`.`role_category` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)

P.S. schema name is caesar

**Comment by kmusienko** [comment]
@vladvorontsov, maybe we should delete lines like "ON DELETE NO ACTION", "ON UPDATE NO ACTION", because MySQL uses these properties by default.

File: src/main/resources/ceasar.sql [hunk 1]
```diff
@@ -0,0 +1,214 @@
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+DROP SCHEMA IF EXISTS `сeasar` ;
+
+CREATE SCHEMA IF NOT EXISTS `сeasar` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci ;
+USE `сeasar` ;
```
Comment (thedikiy) [inline src/main/resources/ceasar.sql]: Found solution for my problem seems like you have russian character 'c' in schema name or something like that

**Comment by vladvorontsov** [comment]
Okey, guys) sorry about this problems, it was working for me, but today i
will find solution of it. Thanks for your review

**Comment by thedikiy** [comment]
@vladvorontsov i made some changes to your script and now it works. And now i'm working on fact that i can't import it with hibernate, having error "Constraint fails", but via Workbench it works fine

**Comment by thedikiy** [comment]
@vladvorontsov Finally it seems i found solution you can check your script in merge_prj branch

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 # hibernate.X
-hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
-hibernate.show_sql=true
-hibernate.hbm2ddl.auto=validate
\ No newline at end of file
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
+spring.jpa.hibernate.ddl-auto=create
+spring.jpa.properties.hibernate.hbm2ddl.import_files_sql_extractor=org.hibernate.tool.hbm2ddl.MultipleLinesSqlCommandExtractor
+spring.jpa.properties.hibernate.hbm2ddl.import_files=caesar.sql,insert.sql
+spring.jpa.properties.hibernate.show_sql=true
```
Comment (svoronin) [inline src/main/resources/application.properties]: If we want those debug options to be checked into master, I'd say to go with those four:
```
spring.jpa.properties.hibernate.show_sql=true
spring.jpa.properties.hibernate.use_sql_comments=true
spring.jpa.properties.hibernate.format_sql=true
spring.jpa.properties.hibernate.type=trace
```

**Comment by svoronin** [comment]
Please resolve conflicts with ```application.properties``` and it will be good to go.

**Pull request merged**